python/felis/db/variants.py

```diff
@@ -28,26 +13,39 @@
 from ..datamodel import Column
 from .dialects import get_dialect_module, get_supported_dialects
 
+__all__ = ["make_variant_dict"]
+
 
 def _create_column_variant_overrides() -> dict[str, str]:
-    """Create a dictionary of column variant overrides."""
+    """Map column variant overrides to their dialect name.
+
+    Notes
+    -----
+    This function is intended for internal use only.
+    """
     column_variant_overrides = {}
     for dialect_name in get_supported_dialects().keys():
         column_variant_overrides[f"{dialect_name}_datatype"] = dialect_name
     return column_variant_overrides
 
 
-_COLUMN_VARIANT_OVERRIDES = _create_column_variant_overrides()
+_COLUMN_VARIANT_OVERRIDES = MappingProxyType(_create_column_variant_overrides())
+"""Map of column variant overrides to their dialect name."""
 
 
-def _get_column_variant_overrides() -> dict[str, str]:
-    """Return a dictionary of column variant overrides."""
+def _get_column_variant_overrides() -> Mapping[str, str]:
+    """Get a dictionary of column variant overrides."""
     return _COLUMN_VARIANT_OVERRIDES
 
 
 def _get_column_variant_override(field_name: str) -> str:
-    """Return the dialect name from an override field name on the column like
```

Comment:
"Return" wasn't allowed?

Reply:
I see this was resolved without comment but I'd still like to know why Return was replaced by Get. Is this because the function is called `_get`?